sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -738,7 +738,7 @@ def _execute_query(self, bq):
         job_id=uuid.uuid4().hex,
         kms_key=self.kms_key)
     job_ref = job.jobReference
-    bq.wait_for_bq_job(job_ref)
+    bq.wait_for_bq_job(job_ref, max_retries=0)
```

Comment:
Probably cleaner to just change to default to wait for ever unless a timeout is specified ?